lib/presenter.rb

```diff
@@ -1,5 +1,5 @@
 class Presenter
-  def display_board(board)
+  def board(board)
```

Comment:
This name feels a little weird to me. Maybe it is because the name and the parameter share the same name?

```ruby
presenter.board(board)
```

Reply:
will adjust